### DISCUSSION TITLE
MMpose in Kaggle with GPU does not work

### DISCUSSION THREAD

**Discussion by flexfalk** [open-mmlab/mmpose#2984] [Q&A]
Hello all, 
I have a problem. When i use MMpose in Kaggle with GPU support i get an error related to the 
mmcv.ops module. 
This error only arises when GPU is enabled. I followed the tutorial completely and have spent many hours of debugging. 
Can you please provide a guide on how to use MMPose with GPU support in Kaggle / Google Colab (Your Guide to Google Colab installation also break when you enable GPU)

**Comment by Ben-Louis** [comment]
Perhaps you could try to install mmcv from source
